### PR TITLE
Add test for ImageBitmap [Serializable] property

### PR DESCRIPTION
**Pull request by Cwiiis**
Addresses issue #19439

### CODE REVIEW

File: 2dcontext/imagebitmap/createImageBitmap-serializable.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>createImageBitmap serialize test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/media.js"></script>
+<script src="/common/namespaces.js"></script>
```
Comment (fserb) [inline 2dcontext/imagebitmap/createImageBitmap-serializable.html]: are those needed?

**Review by fserb — APPROVED** [review]

File: 2dcontext/imagebitmap/createImageBitmap-serializable.html (hunk 1 above)
Comment (Cwiiis) [reply 2dcontext/imagebitmap/createImageBitmap-serializable.html]: Not sure, I derived this test from createImageBitmap-transfer.html

Comment (fserb) [reply 2dcontext/imagebitmap/createImageBitmap-serializable.html]: Sure. Please remove the ones that are not needed.

Comment (Cwiiis) [reply 2dcontext/imagebitmap/createImageBitmap-serializable.html]: Only media.js isn't needed I think, I've removed it - namespaces isn't strictly needed, but is a dependency of common.sub.js below, and I suppose a linter might fail if it isn't included(?)

**Comment by Cwiiis** [comment]
This has passed review and I've addressed the comment, is there anything more needed for this to be merged?

**Pull request merged**